.github/workflows/integration.yml

```diff
@@ -28,6 +28,7 @@ jobs:
             | grep -E '^ ' \
             | grep -v \
             -e 'any_network' \
+            -e 'encoding_dyn_abi' \
```

Comment:
Should not be necessary, this is an exclusionary list of examples that cannot be ran due to certain dependencies - your example should be fine

Reply:
gotcha! removed.